reflex/utils/__init__.py

```diff
@@ -1 +1,2 @@
 """Reflex utilities."""
+DEPRECATED_FEATURES = set()
```

Comment:
Is there a reason to not just define this inside `console.py` itself? I dont think it will be used elsewhere.

I also suggest a different name, something like `_EMITTED_DEPRECATION_WARNINGS`

Reply:
fixed